packages/cursorless-vscode/src/constructTestHelpers.ts

```diff
@@ -61,7 +62,7 @@ export function constructTestHelpers(
     },
 
     setStoredTarget(
-      editor: vscode.TextEditor,
```

Comment:
Is this really an improvement?

Reply:
I made these changes when I thought we'd be using `vscode` as the name of our wrapper, so was planning to start deprecating our use of `import * as vscode from "vscode"` . After renaming it to `vscodeApi`, we prob don't need to deprecate that. Happy to roll these back

Reply:
Ah got it. Please do that :)